### ISSUE TITLE
Does not see body-schema which is not $ref to definitions

### ISSUE THREAD

**Issue by evilsneer** [oliyh/martian#100]
If parameters schema for point in swagger.json described without $ref, but like 

```
{
    "operationId": "codes_create",
    "summary": "codes create",
    "description": "API endpoint that allows code_models to be viewed or edited.",
    "parameters": [
        {
            "name": "data",
            "in": "body",
            "required": true,
            "schema": {
                "type": "object",
                "properties": {
                    "dataset": {
                        "type": "integer"
                    },
                    "description": {
                        "type": "string"
                    },
                    "goal": {
                        "type": "integer"
                    },
                    "name": {
                        "type": "string"
                    },
                    "state": {
                        "type": "integer"
                    },
                    "systems": {
                        "type": "array",
                        "items": {
                            "type": "integer"
                        }
                    }
                }
            }
        }
    ],
    "responses": {
        "201": {
            "description": "Create success response",
            "schema": {
                "$ref": "#/definitions/serialiser-Code"
            }
        }
    },
    "tags": [
        "code"
    ]
}
```

then Martian method schema seems like `{:data s/Any}`

It is still valid swagger, I assume. It perfectly works with swagger-ui.

I guess I could dig somewhere around `martian.swagger/body(?)-schema` to fix it?

**Comment by oliyh** [comment]
Hi,

Thanks for reporting this. Could you show the output of `(martian.core/explore martian :codes_create)` and indicate what you would expect it to look like instead?

Cheers

**Comment by evilsneer** [comment]
It looks like 

```{:summary "Create Code object",
 :parameters {:data {:_ nil}},
 :returns
 {201 RETURN-PARAMS-IS-OK}}```

as you see, `:parameters` is {:data s/ANY}, аt fact.

I expect some kind of 

```{:summary "Create Code object",
 :parameters {:data {
    {:k :dataset} {:schema {:p? #obj :pred-name integer?}}
    {:k :description} {:schema {:p? #obj :pred-name string?}}
    {:k :goal} {:schema {:p? #obj :pred-name integer?}}
    {:k :name} {:schema {:p? #obj :pred-name string?}}
    {:k :state} {:schema {:p? #obj :pred-name integer?}}
    {:k :systems} [{:p? #obj, :pred-name integer?}]}},
 :returns
 {201 RETURN-PARAMS-IS-OK}}```

**Comment by oliyh** [comment]
Hi @evilsneer 

Thanks, I will look into this

**Comment by oliyh** [comment]
Hi @evilsneer 

This should be fixed in the latest `0.1.15-SNAPSHOT` which is now on clojars, could you try it and let me know?

Thanks

**Comment by evilsneer** [comment]
Yes, it works perfectly. Thanks!
